x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/connector/action/RestUpdateConnectorConfigurationAction.java

```diff
@@ -39,7 +40,7 @@ protected RestChannelConsumer prepareRequest(RestRequest restRequest, NodeClient
         return channel -> client.execute(
             UpdateConnectorConfigurationAction.INSTANCE,
             request,
-            new RestToXContentListener<>(channel, UpdateConnectorConfigurationAction.Response::status, r -> null)
+            new RestToXContentListener<>(channel, r -> RestStatus.OK)
```

Comment:
`OK` is the default, you can omit this parameter:

```suggestion
            new RestToXContentListener<>(channel)
```